status: exit status: 0
child output:
	   0: backtrace_apple_no_dsymutil::main
	   1: core::ops::function::FnOnce::call_once
	   2: std::sys_common::backtrace::__rust_begin_short_backtrace
	   3: std::rt::lang_start::{{closure}}
	   4: core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once
	             at /rustc/bd39bbb4bb92df439bf6d85470e296cc6a47ffbd/library/core/src/ops/function.rs:287:13
	   5: std::panicking::try::do_call
	             at /rustc/bd39bbb4bb92df439bf6d85470e296cc6a47ffbd/library/std/src/panicking.rs:483:40
	   6: std::panicking::try
	             at /rustc/bd39bbb4bb92df439bf6d85470e296cc6a47ffbd/library/std/src/panicking.rs:447:19
	   7: std::panic::catch_unwind
	             at /rustc/bd39bbb4bb92df439bf6d85470e296cc6a47ffbd/library/std/src/panic.rs:140:14
	   8: std::rt::lang_start_internal::{{closure}}
	             at /rustc/bd39bbb4bb92df439bf6d85470e296cc6a47ffbd/library/std/src/rt.rs:148:48
	   9: std::panicking::try::do_call
	             at /rustc/bd39bbb4bb92df439bf6d85470e296cc6a47ffbd/library/std/src/panicking.rs:483:40
	  10: std::panicking::try
	             at /rustc/bd39bbb4bb92df439bf6d85470e296cc6a47ffbd/library/std/src/panicking.rs:447:19
	  11: std::panic::catch_unwind
	             at /rustc/bd39bbb4bb92df439bf6d85470e296cc6a47ffbd/library/std/src/panic.rs:140:14
	  12: std::rt::lang_start_internal
	             at /rustc/bd39bbb4bb92df439bf6d85470e296cc6a47ffbd/library/std/src/rt.rs:148:20
	  13: std::rt::lang_start
	  14: main
	  15: <unknown>
	  16: __libc_start_main
	  17: _start
	
	
	
